.gitignore

```diff
@@ -1,6 +1,3 @@
+.phpunit.result.cache
 composer.lock
-phpunit.xml
```

Comment:
See comment on #32 /phpunit.xml is the local version of /phpunit.xml.dist

Reply:
Fixed in #34